RELEASE-NOTES

```diff
@@ -19,7 +19,7 @@ Usage
 -----
 
 When you first run Electron Cash it will use a different configuration directory to Electrum.  On Unix it is
-".electron-cash", and on Windows/MacOS it is "ElectronCash".  Your wallet files will be copied from the Electrum
+".electron-cash", and on Windows/MacOS it is "Vilight".  Your wallet files will be copied from the Electrum
```

Comment:
need to replace ```.electron-cash``` references with ```.vilight```

Reply:
The file paths should be changed to follow system specific file paths. Such as macOS should be in ~/Library/Application Support/Vilight